flink-runtime/src/test/java/org/apache/flink/runtime/operators/hash/MemoryHashTableTest.java

```diff
@@ -100,340 +101,371 @@
 	@Test
 	public void testDifferentProbers() {
 		final int NUM_MEM_PAGES = 32 * NUM_PAIRS / PAGE_SIZE;
-		
-		AbstractMutableHashTable<IntPair> table = new CompactingHashTable<IntPair>(serializer, comparator, getMemory(NUM_MEM_PAGES, PAGE_SIZE));
-		
+		testDifferentProbersCore(new CompactingHashTable<>(serializer, comparator, getMemory(NUM_MEM_PAGES, PAGE_SIZE)), NUM_MEM_PAGES);
```

Comment:
OK, I'll split the top-level methods into two methods, and move these to different classes. However, the core of the tests can remain shared, right?

For example, I would split `testBuildAndRetrieve` to two methods, and move these to different classes, but both of them would still call the same `testBuildAndRetrieveCore`.